spec/requests/topics_controller_spec.rb

```diff
@@ -116,6 +119,30 @@
           expect(Tag.all.pluck(:name)).to contain_exactly("tag1", "tag2")
         end
 
+        it "forces resulting topic owner to watch the new topic" do
```

Comment:
I don't think this should be tested in a request spec. It should instead be tested in `post_mover_spec`.